compose/ts-translation-service.yml

```diff
@@ -0,0 +1,37 @@
+version: '2.4'
+
+services:
+  ts-translation-service:
+    build:
+      context: .
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+    image: "hmctspublic.azurecr.io/ts/translation-service:latest"
+    container_name: translation-service
+    environment:
+      TS_TRANSLATION_SERVICE_DB_HOST: "${CCD_POSTGRES_11:-ccd-shared-database}"
+      TS_TRANSLATION_SERVICE_DB_PORT: 5432
+      TS_TRANSLATION_SERVICE_DB_NAME: ts_translation_service
+      TS_TRANSLATION_SERVICE_DB_USERNAME: ${DB_USERNAME}
+      TS_TRANSLATION_SERVICE_DB_PASSWORD: ${DB_PASSWORD}
+      IDAM_API_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      IDAM_OIDC_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      S2S_URL: http://service-auth-provider-api:8080
+    ports:
+      - "4650:4650"
+    depends_on:
+      ccd-shared-database:
+        condition: service_started
+      # Uncomment this line to enable ccd-shared-database with Postgres version 11
+      #ccd-shared-database-v11:
+      #  condition: service_started
+      service-auth-provider-api:
+        condition: service_started
+    networks:
+      - ccd-network
+
+
+volumes:
+  ts-database-data:
```

Comment:
As the TS database will reuse the ccd-shared-database service I assume this will not be required.

Reply:
done

Reply:
done